gluster_exporter/main.go

```diff
@@ -33,7 +35,24 @@ func registerMetric(name string, fn func(), intervalSeconds int64) {
 }
 
 func getPeerID() string {
-	return *peerid
+	gdInfo, err := os.Open("/var/lib/glusterd/glusterd.info")
```

Comment:
This path is correct for gd1. But for gd2 we need to look at `/var/lib/glusterd2/uuid.toml`.

Reply:
I don't have gd2 installed, so I'll leave this up to someone else to complete.

Reply:
sure. thanks

Reply:
Do you have a copy of what the uuid.toml file should look like?